src/main/java/mate/controller/ShowAllDriversController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+public class ShowAllDriversController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
```suggestion
    protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
in most classes you have protected modifier, check other methods as well. And maybe change, so all methods have same modifier